server/domain/auth.ts

```diff
@@ -134,22 +134,25 @@ export const fetchSessionId = async (
   let sessionId = ''
 
   switch (accountAction) {
-    case 'signUp':
+    case 'signUp': {
```

Comment:
`error  Unexpected lexical declaration in case block` という警告が発生したので修正🐱

おそらく以前からそうだったが、ESLintのpackage更新によって新たに警告を出すようになったみたいだ！

https://mitsuruog.github.io/javascript-style-guide/#comparison--switch-blocks